Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -169,8 +236,8 @@
         return;
     }
 
-    [registeredContentScriptsStore() addScripts:toWebAPI(scripts) completionHandler:makeBlockPtr([this, protectedThis = Ref { *this }, scripts, injectedContentsMap = WTFMove(injectedContentsMap), completionHandler = WTFMove(completionHandler)](NSString *errorMessage) mutable {
-        if (errorMessage) {
+    registeredContentScriptsStore()->addScripts(toWebAPI(scripts), [this, protectedThis = Ref { *this }, scripts = WTFMove(scripts), injectedContentsMap = WTFMove(injectedContentsMap), completionHandler = WTFMove(completionHandler)](const String& errorMessage) mutable {
```

Comment:
Ditto on `WTFMove(scripts)` on the same line as `toWebAPI(scripts)`. Maybe pull `toWebAPI(scripts)` (as `toJSONSerialization`) out into a variable before.